css/main.css

```diff
@@ -17,18 +17,24 @@
   --icon-size: 30px;
 }
 
-html {
-  box-sizing: border-box;
-  overflow:hidden;
-}
-
 *,
 *:before,
 *:after {
   box-sizing: border-box;
   margin: 0;
 }
 
+html {
+  box-sizing: border-box;
+  overflow-x: hidden;
```

Comment:
General overflow was set before, beause otherwise the scrollbar will show when clicking on expand menu (bottom left). This due to the rotating animation of the arrow.